BetterShardsBukkit/src/main/resources/config.yml

```diff
@@ -1,5 +1,28 @@
 mysql:
+ username: bukkit
+ dbname: bukkit
+ password: ''
+ port: 3306
+ host: localhost
  poolsize: 5
  connection_timeout: 10000
  idle_timeout: 600000
- max_lifetime: 7200000
\ No newline at end of file
+ max_lifetime: 7200000
+ 
+locks:
```

Comment:
I guess either populate the defaults inline or add the savedefault() loadconfig() pattern so that this default config file is available.